src/convertToObject.js

```diff
@@ -1,18 +1,21 @@
 'use strict';
 
-/**
- * Implement convertToObject function:
- *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
- * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
- *
- * @param {string} sourceString
- *
- * @return {object}
- */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = {};
+  const sourceStringTrimmed = sourceString.trim();
+  const stringSplit = sourceStringTrimmed.split(';');
+
+  for (const entry of stringSplit) {
+    if (entry.includes(':')) {
+      const index = entry.indexOf(':');
+      const name = entry.slice(0, index).trim();
+      const value = entry.slice(index + 1, entry.length).trim();
```

Comment:
`const splitedEntry = entry.split(':');
const name = splitedEntry[0].trim();
...`

Reply:
It looks simpler, isn't it?)